features/step_definitions/networking.rb

```diff
@@ -989,3 +989,22 @@
   raise "OVN is not running correctly! Check one of your ovnkube-node pod" unless desired_ovnkube_node_replicas == available_ovnkube_node_replicas && available_ovnkube_node_replicas != 0
   raise "OVN is not running correctly! Check one of your ovnkube-master pod" unless desired_ovnkube_master_replicas == available_ovnkube_master_replicas && available_ovnkube_master_replicas != 0
 end
+
+Given /^I enable multicast for the "(.+?)" namespace$/ do | project_name |
+  ensure_admin_tagged
+  _admin = admin
+  @result = _admin.cli_exec(:get, resource: "network.operator", output: "jsonpath={.items[*].spec.defaultNetwork.type}")
+  raise "Unable to find corresponding networkType pod name" unless @result[:success]
+  if @result[:response] == "OpenShiftSDN"
+     @result = admin.cli_exec(:annotate, resource: "netnamespace", resourcename: project_name , keyval: 'netnamespace.network.openshift.io/multicast-enabled=true')
+     unless @result[:success]
+       raise "Failed to apply the default deny annotation to specified namespace."
+     end
+  else
+     @result = admin.cli_exec(:annotate, resource: "namespace", resourcename: project_name , keyval: 'k8s.ovn.org/multicast-enabled=true')
+     unless @result[:success]
+       raise "Failed to apply the default deny annotation to specified namespace."
+     end
+  end  
```

Comment:
it will be somehow less code to do
```
if @result[:response] == "OpenShiftSDN"
  annotation = 'netnamespace.network.openshift.io/multicast-enabled=true'
else
  annotation = 'k8s.ovn.org/multicast-enabled=true'
end
@result = admin.cli_exec(:annotate, resource: "netnamespace", resourcename: project_name , keyval: annotation)
unless @resu...
```

Otherwise looks nice, thank you!

Reply:
@akostadinov  Update the code based your above comments, thanks!
New log from testing ovn cluster:
https://privatebin-it-iso.int.open.paas.redhat.com/?3293d16d87c0a469#y8mtBFCHvDK/b1Jo/whZVH39efe6kfrlACxxMaJs584=

New log from testing sdn cluster:
https://privatebin-it-iso.int.open.paas.redhat.com/?6b88d71539a8b3d4#L0iGQFRRMSjWLdm69TtDKSOU3IhzGIrTwHgFVR8dt6c=